recipes/libxmlsec1/meta.yaml

```diff
@@ -0,0 +1,43 @@
+# Credits to jeanconn for the package https://binstar.org/jeanconn/fftw
+package:
+    name: libxmlsec1
+    version: "1.2.24"
+
+source:
+    fn: xmlsec1-1.2.24.tar.gz
+    url: http://www.aleksey.com/xmlsec/download/xmlsec1-1.2.24.tar.gz
+    md5: 2f4944356b69ea14c18814d9248fbbd8
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+    build:
```

Comment:
this section here should be ``host`` instead of ``build`` and you need to create another section ``build`` with the dependency `` - {{ compiler('cxx') }} ``